src/azure-cli/azure/cli/command_modules/appconfig/keyvalue.py

```diff
@@ -172,23 +173,18 @@ def export_config(cmd,
     src_kvs = __read_kv_from_config_store(
         cmd, name=name, connection_string=connection_string, key=key, label=label, prefix_to_remove=prefix)
 
+    if skip_keyvault:
+        src_kvs = [keyvaule for keyvaule in src_kvs if keyvaule.content_type != KeyVaultConstants.KEYVAULT_CONTENT_TYPE]
```

Comment:
Seems `keyvaule` is a typo here, do you mean `keyvault` or `keyvalue`?

Reply:
Typo. Should be keyvalue